code/code/cmd/cmd_deathstroke.cc

```diff
@@ -62,69 +62,113 @@ static int deathstroke(TBeing *caster, TBeing *victim)
 
   int dam = caster->getSkillDam(victim, SKILL_DEATHSTROKE, lev, caster->getAdvLearning(SKILL_DEATHSTROKE));
 
-  if ((caster->bSuccess(bKnown+ percent, SKILL_DEATHSTROKE) &&
-         (i = caster->specialAttack(victim,SKILL_DEATHSTROKE)) && 
-         (i != GUARANTEED_FAILURE)) ||
-      !victim->awake()) {
+  if (caster->bSuccess(bKnown+percent, SKILL_DEATHSTROKE) || !victim->awake()) {
     wasSuccess = TRUE;
+
     if (!dam) {
       act("$n's attempt at $N's vital area falls far short of hitting.",
                 FALSE, caster, 0, victim, TO_NOTVICT);
       act("You fail to hit $N's vital area.", FALSE, caster, 0, victim, 
                 TO_CHAR);
       act("$n attempts to hit your vital area, but fails miserably.", 
                 FALSE, caster, 0, victim, TO_VICT);
+
+      // You failed to deal damage - assessing a penalty
+      aff.type = SKILL_DEATHSTROKE;
+      aff.duration = Pulse::UPDATES_PER_MUDHOUR / 3;
+      aff.location = APPLY_ARMOR;
+      aff.modifier = caster->GetMaxLevel()*8 - caster->getSkillLevel(SKILL_DEATHSTROKE);
+      aff.bitvector = 0;
     } else {
       act("$n hits $N in $S vital organs!", FALSE, caster, 0, victim, 
                 TO_NOTVICT);
       act("You hit $N in $S vital organs!", FALSE, caster, 0, victim, 
                 TO_CHAR);
       act("$n hits you in your vital organs!", FALSE, caster, 0, 
                 victim, TO_VICT);
+
+      // Adding a lockout
+      // You successfully landed your deathstroke - assessing a bonus
+      aff.type = SKILL_DEATHSTROKE;
+      aff.duration = Pulse::UPDATES_PER_MUDHOUR / 3;
+      aff.location = APPLY_HITNDAM;
```

Comment:
I'm not sure APPLY_HITNDAM works. Did you test how this affect works? Also, 15 sounds insane... at least for APPLY_HIT each +1 is like a levels worth of offense.

Reply:
Ok so hitndam works. My confusion is that it's not something you can apply to eq but it does work.

Here's how the apply affects a char:
https://github.com/sneezymud/sneezymud/blob/master/code/code/sys/handler.cc#L452-L454
so +15 hit == +150 hitroll

then in combat: https://github.com/sneezymud/sneezymud/blob/master/code/code/misc/combat.cc#L2499

so the 150 hitroll turns into 250 offense. You should be able to verify by stating the PC or mob where this is applied.

I think something like +3 HITNDAM is more appropriate.